src/clients/UBAClient/UBAClientAbstract.ts

```diff
@@ -89,7 +90,7 @@ export abstract class BaseUBAClient {
    * @param refundChainId The chainId of the refund
    * @returns The balancing fee for the given token on the given chainId at the given block number
    */
-  protected async computeBalancingFee(
+  public async computeBalancingFee(
```

Comment:
It seems wasteful to pass in both deposit and refund chain id. Why not require user to pass in a single chain, and we assume that that is the deposit chain for a deposit and a refund for a refund?